assets/js/buttons-examples.js

```diff
@@ -10,8 +10,13 @@ $(document).ready(function () {
     "yellow-btn",
     "orange-btn",
     "purple-btn",
+    "denim-btn",
```

Comment:
Why did you add the buttons here? These buttons are just different colors to be used for all button types. Please add them in the `buttons` variable instead under "Special"